README.md

```diff
@@ -1,22 +1,30 @@
 # Viewing Party
+ (include code style and build status)
+ (project overview)
+ (motivations. The Why!)
 
-This is the base repo for the [viewing party project](https://backend.turing.io/module3/projects/viewing_party) used for Turing's Backend Module 3.
+## Demo
+(screenshots of application, heroku?)
 
-### About this Project
+## Tech Framework Used
 
-Viewing party is an application in which users can explore movie options and create a viewing party event for the user and friend's.
+## Features
 
-## Local Setup
+## Code Snippet
 
-1. Fork and Clone the repo
-2. Install gem packages: `bundle install`
-3. Setup the database: `rails db:create`
+### Schema
```

Comment:
We are pretty close to being able to upload this right now - however, I agree with you and think we should wait for a finalized schema until the end? Or should we upload the temporary for now?

Reply:
I like your schema image that we were reviewing.  I think once we walked through how we would maintain a database where one column was an array was finally ironed out, I felt more comfortable with your idea.  Let's post it and if we change, we can update our image with a before and after photo and simply show it as part of our learning.

Reply:
Sounds good! Can be activity for our standup tomorrow, we can continue working through initial setup and databasing first!